tests/Sniffs/Functions/data/unusedParameterNoErrors.php

```diff
@@ -29,6 +29,11 @@ function ($closureSuppress)
 
 };
 
+function usedParameterSupressedViaName($_)
```

Comment:
TODO add another param to confirm that `$_` isn't magical and that `$_foo` is also allowed.